### ISSUE TITLE
Support Paydirect

### ISSUE THREAD

**Issue by cmb69** [cmsimple-xh/xhshop#181]
See https://www.paydirekt.de/haendler/merchant-api.html.

**Comment by frase-git** [comment]
Fraglich, ob sich das lohnt - bzw. sich durchsetzen wird.
[Lesen](https://www.heise.de/newsticker/meldung/Paydirekt-Datenschuetzer-pruefen-Komfortregistrierung-bei-den-Sparkassen-3810710.html)

**Comment by cmb69** [comment]
Ja, das hatte ich gelesen, und dachte wir könnten ja etwas helfen, das das System mehr Verbreitung erlangt, auch ohne solche Aktionen. Scheint zumindest eine interessante Alternative zu Paypal.

**Comment by frase-git** [comment]
> ... und dachte wir könnten ja etwas helfen, das das System mehr Verbreitung erlangt ...

Da bin ich gerne dabei ... soll ich dir meine Kontonummer durchgeben - für die Prozente?
;-)))

Sparkassen und einige Privatbanken: Könnte man das _propietär_ nennen?

**Comment by cmb69** [comment]
> Sparkassen und einige Privatbanken: Könnte man das propietär nennen?

Proprietär gewiss, aber das ist Paypal erst recht.

**Comment by XHalbert** [comment]
> Fraglich, ob sich das lohnt - bzw. sich durchsetzen wird.

Tja wer kann schon in die zukunft blicken?

Fest steht, dass nicht jeder paypal hat oder will, und dem auch nicht vertraut. Ein Sparkassen-Kunde dürfte wohl einem Sparkassen-Bezahlsystem eher trauen.
In dem Artikel wurden Datenschutzbedenken bei der bequemeren Registrierung behandelt - und die werden wie ich glaube gelöst werden.

Also ich fänd paydirekt sinnvoll zu integrieren
